utils.sh

```diff
@@ -56,6 +63,58 @@ function patch_node_ignition() {
     ssh < "${wd}/${kind}.yaml" "${ssh_opts[@]}" "core@$bootstrap" sudo dd of="$config"
 }
 
+function machineconfig_generate_patches() {
+    set -x
+    local kind
+    local path
+    local value
+    local search_path
+    local config_type
+    local files_template
+    local units_template
+    local octal_mode
+    local decimal_mode
+    local dest_path
+    local ifs
+    local simple_name
+
+    read -r -d '' files_template <<'EOF'
+{"op": "add", "path": "${path}/-", "value": {"filesystem": "root", "path": "${dest_path}", "user": {"name": "root"}, "contents": {"source": "data:,${value}", "verification": {}}, "mode": ${decimal_mode}}}
```

Comment:
```suggestion
{"op": "add", "path": "\${path}/-", "value": {"filesystem": "root", "path": "\${dest_path}", "user": {"name": "root"}, "contents": {"source": "data:,\${value}", "verification": {}}, "mode": \${decimal_mode}}}
```
This should protect the values from being evaluated here